mdpow/templates/bar_charmm.mdp

```diff
@@ -1,7 +1,7 @@
 ; Gromacs mdp file
-; Template for MD Hydration FEP with OPLS-AA (part of POW)
+; Template for MD Hydration FEP with CHARMM (part of MDPOW)
```

Comment:
I would put "Solvation" instead of "Hydration".

Reply:
good idea, changed